### PR TITLE
XAMARIN-44068 : Fixed the indent issues in Xamarin

### PR DESCRIPTION
**Pull request by aravindkannanarayanarao**
-Corrected code snippet indent issues in 5 controls

### CODE REVIEW

File: Xamarin-iOS/SfImageEditor/shapes.md [hunk 1]
```diff
@@ -76,10 +78,16 @@ N> The `FillColor` property is applicable only if the ShapeType is [`Rectangle`]
...
 
+{% endcapture %}
+
+{{ codesnippet1 | UnOrderList_Indent_Level_1 }} 
+
 ![SfImageEditor](ImageEditor_images/Shapes.jpg)
 
  * You can annotate any path on an image by using free hand drawing as shown in the below code,
 
+{% capture codesnippet1 %}
```
Comment (vigneshrameshsync) [inline Xamarin-iOS/SfImageEditor/shapes.md:89]: codesnippet1 -> codesnippet2

File: Xamarin-iOS/SfCircularGauge/Ranges.md [hunk 2]
```diff
@@ -94,10 +96,16 @@ The range can be placed inside the scale, outside the scale, or on the scale by
 
 {% endhighlight %}
 
+{% endcapture %}
+
+{{ codesnippet1 | UnOrderList_Indent_Level_1 }}
```
Comment (vigneshrameshsync) [inline Xamarin-iOS/SfCircularGauge/Ranges.md:101]: UnOrderList_Indent_Level_1 -> OrderList_Indent_Level_1

File: Xamarin-iOS/SfCircularGauge/Ranges.md [hunk 3]
```diff
@@ -118,6 +126,10 @@ The range can be placed inside the scale, outside the scale, or on the scale by
 
 {% endhighlight %}
 
+{% endcapture %}
+
+{{ codesnippet2 | UnOrderList_Indent_Level_1 }}
```
Comment (vigneshrameshsync) [inline Xamarin-iOS/SfCircularGauge/Ranges.md:131]: UnOrderList_Indent_Level_1 -> OrderList_Indent_Level_1

File: Xamarin-iOS/SfImageEditor/shapes.md (hunk 1 above)
Comment (aravindkannanarayanarao) [reply Xamarin-iOS/SfImageEditor/shapes.md:89]: Updated above changes

File: Xamarin-iOS/SfCircularGauge/Ranges.md (hunk 2 above)
Comment (aravindkannanarayanarao) [reply Xamarin-iOS/SfCircularGauge/Ranges.md]: updated above changes

File: Xamarin-iOS/SfCircularGauge/Ranges.md (hunk 3 above)
Comment (aravindkannanarayanarao) [reply Xamarin-iOS/SfCircularGauge/Ranges.md]: updated above changes

**Review by vigneshrameshsync — APPROVED** [review]
The corrections are fine.

**Pull request merged**